js/pages/checks/style.css

```diff
@@ -0,0 +1,34 @@
+.warnings .fa-info {
```

Comment:
Why component placed at pages rather than components dir? Looks like we can make it a true component formatted as other components

Reply:
fixed